### DISCUSSION TITLE
Where can I get the stored OAuth tokens from?

### DISCUSSION THREAD

**Discussion by aashutoshrathi** [superfell/SoqlX#155] [Q&A]
When logged in the application with an OAuth Token, how can I retrieve it back for other requests?

**Comment by superfell** [comment]
The refresh tokens are stored in the keychain. But they are tied to the client id. so you can't used them with other applications.

**Comment by aashutoshrathi** [comment]
UPDATE: I was able to pull `accessToken` and `refreshToken` both out by Proxyman
